src/includes/configuration/config-intro/dotnet.google-cloud-functions.mdx

```diff
@@ -0,0 +1,36 @@
+Options can be set on `appsettings.json` or on a class inheriting from `FunctionsStartup`.
```

Comment:
It's not clear here that these are two alternatives.
Either provide values through `appsettings.json` OR provide the `CustomOptions` class.

Also, I don't see a usage of `[assembly: FunctionsStartup(typeof(SentryStartup))]`
Is this going to init the SDK?


See the sample: https://github.com/getsentry/sentry-dotnet/blob/main/samples/Sentry.Samples.Google.Cloud.Functions/Function.cs